docs/standard/threading/overview-of-synchronization-primitives.md

```diff
@@ -1,174 +1,167 @@
 ---
-title: "Overview of Synchronization Primitives"
-ms.date: "03/30/2017"
+title: "Overview of synchronization primitives"
+description: "Learn about .NET thread synchronization primitives used to synchronize access to a shared resource or control thread interaction"
+ms.date: "09/20/2018"
 ms.technology: dotnet-standard
 helpviewer_keywords: 
   - "synchronization, threads"
-  - "threading [.NET Framework],synchronizing threads"
+  - "threading [.NET],synchronizing threads"
   - "managed threading"
 ms.assetid: b782bcb8-da6a-4c6a-805f-2eb46d504309
 author: "rpetrusha"
 ms.author: "ronpet"
 ---
-# Overview of Synchronization Primitives
-<a name="top"></a> The .NET Framework provides a range of synchronization primitives for controlling the interactions of threads and avoiding race conditions. These can be roughly divided into three categories: locking, signaling, and interlocked operations.  
-  
- The categories are not tidy nor clearly defined: Some synchronization mechanisms have characteristics of multiple categories; events that release a single thread at a time are functionally like locks; the release of any lock can be thought of as a signal; and interlocked operations can be used to construct locks. However, the categories are still useful.  
-  
- It is important to remember that thread synchronization is cooperative. If even one thread bypasses a synchronization mechanism and accesses the protected resource directly, that synchronization mechanism cannot be effective.  
```

Comment:
@rpetrusha I've brought it back, but considerably reworked. Instead of focusing on threads, which is difficult to do with the new task-based approach where threading is a low-level implementation detail, the new phrasing focuses on how a resource is accessed, which seems easier to evaluate and implement. 

Related commit: https://github.com/dotnet/docs/pull/7790/commits/fcc5daf41039f573eb101c58302616708bf64f07